### PR TITLE
Re introduce fulltext search

### PR DESCRIPTION
**Pull request by bilalesi**
fixes:  [4723](https://app.zenhub.com/workspaces/blue-brain-nexus-60ace035034cae00105c9307/issues/gh/bluebrain/nexus/4723)
1. add `cmdk` component for fulltext search.
2. use "ctrl + e" to open the command pallete or click on the button in the header
3. use resources list delta endpoint to fetch the results of the query.
4. group the result per org/project in the result component.

## Description
Screenshot 2024-05-28 at 12 12 49
Screenshot 2024-05-28 at 12 13 07
Screenshot 2024-05-28 at 12 13 34

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added necessary unit and integration tests.
- [ ] I have added screenshots (if applicable), in the comment section.

### CODE REVIEW

**Comment by danburonline** [comment]
CleanShot 2024-03-11 at 4  05 04

If you have dark mode enabled, the colours look quite off. Have you tested it in dark mode?

**Comment by danburonline** [comment]
Also, another feedback: Currently, when you're in an input field and type something with a `/`, you always open the command window. Shouldn't we change the shortcut to `CMD + /`? Most apps have it like this.

**Comment by bilalesi** [comment]
> If you have dark mode enabled, the colours look quite off. Have you tested it in dark mode?

we never take dark mode into consideration, the whole app built with light mode in mind,
This is a discussion for future

**Comment by bilalesi** [comment]
> Also, another feedback: Currently, when you're in an input field and type something with a `/`, you always open the command window. Shouldn't we change the shortcut to `CMD + /`? Most apps have it like this.

I changed this to `ctrl + e` and will see the endusers feedback

**Review by danburonline — APPROVED** [review]

File: src/shared/components/FullTextSearch/useFullTextSearch.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const { isLoading, data } = useQuery({
+    enabled: !!search,
+    queryKey: ['cmdk-search', { search }],
+    queryFn: () =>
+      nexus.Resource.list(undefined, undefined, {
+        q: search,
+        deprecated: false,
+      }),
+    select: data => data._results,
+    staleTime: 2,
```
Comment (Dinika) [inline src/shared/components/FullTextSearch/useFullTextSearch.ts:23]: The `staleTime` in milliseconds right? 2 milliseconds sounds too low to me. Did you mean 2000 ms instead?

**Review by Dinika — CHANGES_REQUESTED** [review]
lgtm. posted a question.

**Pull request merged**